px-data-grid-action-column.html

```diff
@@ -94,6 +94,24 @@
           };
         }
 
+        static get observers() {
+          return [
+            '_toggleEditClassName(edittedItem)'
+          ];
+        }
+
+        _toggleEditClassName(edittedItem) {
+          if (this._cells) {
+            this._cells.forEach(cell => {
+              if (edittedItem) {
+                cell.removeAttribute('hidden-unless-hover');
+              } else {
+                cell.setAttribute('hidden-unless-hover', '1');
```

Comment:
What is `'1'`? Why don't just use booleans and check
```
[part~="row"] [part~="body-cell"][hidden-unless-hover]
```
in css?

Reply:
Fixed in ...theme.html, thanks.